depends/patches/zeromq/fix_have_windows.patch

```diff
@@ -0,0 +1,62 @@
+commit b163fc36a48ecdc87a3ecb4c6bba5f6b8965acaf
```

Comment:
Cannot find b163fc36a48ecdc87a3ecb4c6bba5f6b8965acaf in https://github.com/zeromq/libzmq. Did you mean https://github.com/zeromq/libzmq/commit/aa885c5a154256612108636b0fb22f44ae0e247a?

Perhaps we could refer to a PR number, as is done in other patches?